falcon.go

```diff
@@ -36,96 +45,131 @@ const (
 	// CurrentSaltVersion is the salt version number used to compute signatures.
 	// The salt version is incremented when the signing procedure changes (rarely).
 	CurrentSaltVersion = C.FALCON_DET1024_CURRENT_SALT_VERSION
+	// CTSignatureSize is the max size in bytes of a Falcon signature in CT format
+	CTSignatureSize = C.FALCON_DET1024_SIG_CT_SIZE
```

Comment:
this is interesting, I didn't know you could set Go consts from C macro expressions. How does it know what type to use?

Reply:
I think it translates into a number.
so the default is int
you could do `CTSignatureSize = uint64(C.FALCON_DET1024_SIG_CT_SIZE)`

Reply:
magic. I guess you could set a type for these constants if you wanted to ensure the type, but LGTM otherwise